src/plugins/action_files_cleanup/src/plugin.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "action_plugin",
+  "code": "cleanupFiles",
+  "name": "Clean up application files",
+  "file": "cleanupFiles",
+  "function_name": "cleanupFiles",
+  "description": "Checks all the uploaded files associated with an application, and deletes any that are not part of an application_response",
+  "required_parameters": ["applicationId"],
```

Comment:
i think these are optional parameters (also has applicationSerial).

Btw I would suggest not using 'serial' (apart from some UI and URL in front end) <- I still think we should have just used the id of application in URL, and use serial for display purposes only.

The serial to id change would require quite a bit of juggling, so not a required change, just something to think about for future cases.

Reply:
Yeah, I think it's useful to have "serial" available in `applicationData` for some situations. Ideally Actions could take either and still work fine. But sure, ID as the main one to use for back-end makes sense (except for when it doesn't lol).